### ISSUE TITLE
Creopyson Drawing Create Error

### ISSUE THREAD

**Issue by PiNandanwar** [Zepmanbc/creopyson#40]
* Creo version: 6.0.6
* Creopyson version: 0.7.2
* Creoson version:  2.8
* Python version: 3.8 (accidentally have a 32 bit version)
* Operating System: Windows 10 64  bit

I am trying to create a drawing using Creopyson and keep getting a very generic error. I am able to run the Creopyson server and connect through Python to an open Creo session. I am able to do a few model specific functions (see active parts, get family table instances, open family table instance parts, etc) but the drawing functions are proving to be a bit more difficult.

I created a brand new part that has no features and am trying to create a drawing for it as a test before writing more code. 

I have the new part as the active part in my Creo session and then I run the code and get a  'A Pro/TOOLKIT error has occurred: General Error.  Check your model/drawing for accuracy.' error.

I am also connected to Windchill, not sure if that could be causing the issue. 

I also noticed that if i do not have the .frm for the template that Creopyson will throw an error that it can't find the format. 

My code is as follows:

```
import creopyson as cps
import os.path
import logging

logging.basicConfig(level = logging.DEBUG)
logging.getLogger("urllib3").setLevel(logging.WARNING)

c = cps.Client()
c.connect()
c.server
sId = c.sessionId

active_part = cps.file.get_active(c)

formatName = "FLAT.FRM"

c.drawing_create(formatName)
```
I turned on debugging so I could see what was going on, but am still at a loss for what exactly the issue is.

```
DEBUG:creopyson.connection:request: {'sessionId': '', 'command': 'connection', 'function': 'connect', 'data': None}
DEBUG:creopyson.connection:response: {'status': {'error': False}, 'sessionId': '-8041623088975976175', 'data': None}
DEBUG:creopyson.connection:request: {'sessionId': '-8041623088975976175', 'command': 'file', 'function': 'get_active', 'data': None}
DEBUG:creopyson.connection:response: {'status': {'error': False}, 'data': {'file': 'adsfasdf.prt', 'dirname': 'C:/ProETemp/'}}
DEBUG:creopyson.connection:request: {'sessionId': '-8041623088975976175', 'command': 'drawing', 'function': 'create', 'data': {'template': 'flat.frm'}}
DEBUG:creopyson.connection:response: {'status': {'error': True, 'message': 'A Pro/TOOLKIT error has occurred: General Error.  Check your model/drawing for accuracy.'}, 'data': None}
```

**Comment by Zepmanbc** [comment]
Hello

the format name is not the "frm" file name, you should use the name like here (sorry for the french screenshot but I think you will find)
![image](https://user-images.githubusercontent.com/10487746/128315339-43ba4306-82d6-4692-8d0b-019fbcee1640.png)
here is a sample of code:
```
import creopyson

c = creopyson.Client()
c.connect()

c.creo_cd(r"C:\Users\Zepman\3D Objects\CAO\creopyson_demo")
listfiles = c.creo_list_files("*.prt")
c.file_open(listfiles[0])
view_list = c.view_list()
if "FRONT" in view_list:
    c.drawing_create("a4_drawing", display=True, activate=True)
```
I've never used this feature before, I had some troubles with a missing view needed in the template, You should do some tests before creating your drawing because there is no specific error rising. 
I had some troubles if *display* and *activate* are **False**, maybe I should put default to **True**. 
Let me know from your own experience

**Comment by PiNandanwar** [comment]
Zepmanbc, 

That fixed it! Thanks for the quick reply, I'll go ahead and close this issue out. 

Also, I did play around with the _display_ and _activate_ being **False**, and it seems like the drawing is created in session but then throws a General Error right after that step. I was able to get a few other functions written after drawing creation but those did not run when those variables were set to **False**. 

Best combination was using **True** for _display_ and _activate_ and also adding _new_window_ and setting that to **True**. I'm guessing it would be easy enough to just close the drawing after creating and doing whatever to it, at least for what I need to do currently.
